docs/metrics/learning-prometheus-server/README.md

```diff
@@ -0,0 +1,184 @@
+  - [Get Prometheus](#get-prometheus)
+  - [Prometheus server](#prometheus-server)
+    - [Configuration](#configuration)
+    - [Start Prometheus](#start-prometheus)
+    - [Configure OpenTelemetry to Expose metrics to Prometheus Endpoint](#configure-opentelemetry-to-expose-metrics-to-prometheus-endpoint)
+    - [Check Results in Prometheus](#check-results-in-prometheus)
+    - [View/Query Results with Grafana](#viewquery-results-with-grafana)
+
+## Prerequisite
+
+It is highly recommended to go over the [getting-started](../getting-started/README.md)
+project under the metrics document folder before following along this document.
+
+## Introduction
+
+- [What is Prometheus?](https://prometheus.io/docs/introduction/overview/)
+
+- [Grafana support for
+  Prometheus](https://prometheus.io/docs/visualization/grafana/#creating-a-prometheus-graph)
+
+## Get Prometheus
+
+Follow the [first steps]((https://prometheus.io/docs/introduction/first_steps/))
+in the prometheus official document.
+
+## Prometheus server
+
+### Configuration
+
+After downloading the [latest release](https://prometheus.io/download/), extract
+it to a local location that's easy to access. We will find the default
+Prometheus configuration YAML file in the folder, named `prometheus.yml`.
+
+Let's create a new file in the same location as `prometheus.yml`, called
+`otel.yml` for this exercise. Copy and paste the entire content below into the
+otel.yml file we have created just now.
+
+```yaml
+global:
+  scrape_interval: 10s
+  scrape_timeout: 10s
+  evaluation_interval: 10s
+scrape_configs:
+- job_name: MyOpenTelemetryDemo
+  honor_timestamps: true
+  scrape_interval: 1s
+  scrape_timeout: 1s
+  metrics_path: /metrics
+  scheme: http
+  follow_redirects: true
+  static_configs:
+  - targets:
+    - localhost:9184 # set the port that prometheus server will listen to
+```
+
+### Start Prometheus
+
+Follow the instructions from
+[starting-prometheus](https://prometheus.io/docs/introduction/first_steps/#starting-prometheus)
+to start the Prometheus server and verify it has been started successfully.
+
+Please note that we will need pass in otel.yml file as the argument:
+
+```console
+./prometheus --config.file=otel.yml
+```
+
+Once the server is started, we are going to make some small tweaks to the
+example in the getting-started metrics [example](../getting-started/Program.cs)
+to export our metrics to the endpoint that prometheus was configured to listen
+to.
+
+### Configure OpenTelemetry to Expose metrics to Prometheus Endpoint
+
+Create a new console application and run it:
+
+```sh
+dotnet new console --output prometheus-http-server
+cd prometheus-http-server
+dotnet run
+```
+
+We will have to add a reference to prometheus exporter to the
+prometheus-http-server application.
+
+```shell
+dotnet add package OpenTelemetry.Exporter.Prometheus --version 1.2.0-rc1
+```
+
+Now, replace the below line in the [Program.cs](../getting-started/Program.cs)
```

Comment:
since we just created a new console app above, it won't have AddConsoleExporter line in it. I'd suggest to avoid cross referencing the getting-started/program here.

Reply:
I've added a step for copy and paste from the metrics getting-started Program.cs file step to make it right!